AnnoyancesFilter/sections/popups.txt

```diff
@@ -106,6 +106,7 @@ zsa.io##.gehvRn
 newyorker.com##.paywall-registration-gate-container
 ||misumi-ec.com/operation/floating_bnr/
 penguinrandomhouse.com##.sgnup
+test.de##.page > div.js-accessible-dialog
```

Comment:
I can't reproduce it on my end with German IP. Please attach a screenshot with this element from devtools.

Reply:
Stopped reproducing on my side too.